nixos/modules/services/audio/alsa.nix

```diff
@@ -3,132 +3,38 @@
 
 with lib;
 
-let
-
-  inherit (pkgs) alsa-utils;
```

Comment:
@K900 Speaking as one of the two people still running ALSA, setting `hardware.alsa.enablePersistence` to `true` gives me an `undefined variable 'alsa-utils'` error upon evaluation—I suppose because this line was removed? Or am I doing something wrong?

Reply:
Yes, sorry. Please test https://github.com/NixOS/nixpkgs/pull/327290